qiskit/circuit/library/templates/rzx/rzx_zz2.py

```diff
@@ -34,7 +21,21 @@
 
 
 def rzx_zz2(theta: ParameterValueType | None = None):
-    """Template for CX - RZGate - CX."""
+    """RZX-based template for CX - RZGate - CX.
```

Comment:
```suggestion
    """RZX-based template for CX - PhaseGate - CX.
```

The module-level docstring of this one (correctly) says `CX - PhaseGate - CX`, but the function-level one had an existing copy-paste error.